package.json

```diff
@@ -0,0 +1,69 @@
+{
+  "name": "hof-frontend-assets",
+  "version": "0.1.0",
+  "main": "./bin/index.js",
+  "description": "Frontend JS, CSS and images for a form service",
+  "license": "GPLv3",
+  "author": "UKHomeOfficeForms",
+  "repository": "https://github.com/UKHomeOfficeForms/hof-frontend-assets.git",
+  "bugs": "https://github.com/UKHomeOfficeForms/hof-frontend-assets/issues",
+  "bin": {
+    "hof-frontend-assets": "./bin/index.js"
+  },
+  "scripts": {
+    "test": "npm-run-all lint style browser:test unit:coverage",
+    "browser:test": "karma start --single-run",
+    "unit:test": "mocha test/unit",
+    "cover": "istanbul cover _mocha -- test/unit",
+    "check-coverage": "istanbul check-coverage",
+    "unit:coverage": "npm-run-all cover check-coverage",
+    "snyk": "snyk test",
+    "snyk:dev": "snyk test --dev",
+    "lint": "eslint .",
```

Comment:
`eslint` is not listed in the dependencies. Using with `npm@3` it still "works" becuase it finds the eslint installed as a child of `eslint-config-homeoffice`, but with `npm@2` this is not flattened to the top level, and so fails.

See https://travis-ci.org/UKHomeOfficeForms/hof-frontend-assets/jobs/188810941